### PR TITLE
AlienFlight F4 support

### PR DESCRIPTION
**Pull request by MJ666**
Initial AlienFlight F4 support
Some AlienFlight F3 updates and fixes

### CODE REVIEW

File: src/main/fc/fc_init.c [hunk 1]
```diff
@@ -168,30 +169,34 @@ void init(void)
 
     printfSupportInit();
 
-    initEEPROM();
```
Comment (MJ666) [inline src/main/fc/fc_init.c:171]: The order changes in this area need testing for not having any negative impact.

**Comment by MJ666** [comment]
AlienFlight F7 support will also follow.

**Comment by digitalentity** [comment]
Looks good in general, however init order changes require some testing on other targets.

**Comment by MJ666** [comment]
Rebased and documentation update.

**Comment by MJ666** [comment]
Updated and rebased today.

**Comment by MJ666** [comment]
The AlienFlight F4 target passed all flight tests at an 110mm brushed quadcopter during the last days. I have not seen any issues. Its flying quite well but not as agile as BF.
